xinference/client/restful/restful_client.py

```diff
@@ -535,11 +535,60 @@ def inpainting(
         response_data = response.json()
         return response_data
 
-    def ocr(self, image: Union[str, bytes], **kwargs):
+    def ocr(
+        self,
+        image: Union[str, bytes],
+        prompt: str = "<image>\nFree OCR.",
```

Comment:
Ditto, this is too specific, I think we don't need to change the APIs, just put everything else into kwargs.